### PR TITLE
Updating documentation for custom claim headers

### PR DESCRIPTION
**Pull request by allupaku**
This is for updating changes made via pull request https://github.com/keycloak/keycloak-gatekeeper/pull/532

### CODE REVIEW

**Comment by allupaku** [comment]
This is to be merged with the pull request 
https://github.com/keycloak/keycloak-gatekeeper/pull/532

File: tests/src/test/resources/ignored-links [hunk 1]
```diff
@@ -33,4 +33,5 @@ https://raw.githubusercontent.com/jboss-container-images/redhat-sso-7-openshift-
 https://issues.redhat.com/secure/CreateIssueDetails*
 https://developer.paypal.com/developer/applications
 https://account.live.com/developers/applications/create
-https://developer.twitter.com/apps/
\ No newline at end of file
+https://developer.twitter.com/apps/
+https://docs.jboss.org/author/display/WFLY10/Admin+Guide*
```
Comment (stianst) [inline tests/src/test/resources/ignored-links]: We need to fix these URLs rather than ignore them

File: release_notes/topics/8_0_0.adoc [hunk 2]
```diff
@@ -85,7 +85,7 @@ The option to provide client credentials in the JavaScript adapter was removed.
 
 * Secure token and logout endpoint were included in Gatekeeper. Thanks to https://github.com/fredbi[fredbi]
 * There was a bug on Gatekeeper which was making cookies to be applied to subdomains. Thanks to https://github.com/daniel-ac-martin[daniel-ac-martin] the issue was fixed
-* Now Gatekeeper provides support to https://www.owasp.org/index.php/SameSite[Same-site cookies]. Thanks to https://github.com/fiji-flo[fiji-flo]
+* Now Gatekeeper provides support to https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute[Same-site cookies]. Thanks to https://github.com/fiji-flo[fiji-flo]
```
Comment (stianst) [inline release_notes/topics/8_0_0.adoc:88]: We should just remove this link

**Comment by stianst** [comment]
@abstractj didn't we move the docs? Or plan to? If so docs PRs should be sent to Louketo repo right?

File: release_notes/topics/8_0_0.adoc (hunk 2 above)
Comment (allupaku) [reply release_notes/topics/8_0_0.adoc:88]: Because we dont support the feature any more ? samesite cookie attribute ?

**Comment by abstractj** [comment]
> @abstractj didn't we move the docs? Or plan to? If so docs PRs should be sent to Louketo repo right?

@stianst yes, we moved the docs to Louketo. I just wonder if we should update the Keycloak documentation, by removing the Gatekeeper section and pointing people to Louketo. Wdyt?

**Comment by abstractj** [comment]
@allupaku thanks for contributing to this. Although, Louketo reache it's EOL and we no longer support it. For this reason, I'm closing the issues here.

**Pull request closed without merging**